README.md

```diff
@@ -86,26 +93,50 @@ services:
 
 #### Kubernetes
 
-If you wanted to run this on Kubernetes as a sidecar for monitoring specific services, you can follow the instructions below which use Zookeeper as an example.
+If you wanted to run this on Kubernetes as a sidecar for monitoring specific services, you can follow the instructions below which use Apache Server as an example. This setup the agent will monitor only services running in particular pod(s), not Kubernetes itself.
```

Comment:
maybe: 
In this setup, the agent will monitor

Reply:
Fixed